lorien/InfiniteCanvas/InfiniteCanvas.gd

```diff
@@ -46,21 +46,23 @@ func _ready():
 	_viewport.size = OS.window_size
 
 # -------------------------------------------------------------------------------------------------
-func _input(event: InputEvent) -> void:
+func _gui_input(event: InputEvent) -> void:
```

Comment:
I wanted to use `_unhandled_event` here, but because InfiniteCanvas is a `Control` `_gui_input` is as good as it gets without more complicated code restructuring. Good enough for me though.